moviesandstars/models/movie.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const MovieSchema = new Schema({
+  title : String,
+  genre : String,
+  plot : String,
+})
+
+const Movie = mongoose.model('Movie', MovieSchema);
+module.exports = Movie;
```

Comment:
#checked. You're pushing the node_modules. Next time put them in the gitignore file.